maint/requirements.txt

```diff
@@ -1,12 +1,18 @@
+#
+# This file is autogenerated by pip-compile with python 3.10
+# To update, run:
+#
+#    pip-compile --no-annotate
```

Comment:
I think we should either add pip-compile to requirements.in or remove these comments before checking the file in.